### PR TITLE
Docs: 229 update contributing

### LINKED ISSUE #229: Docs: Add link to zulip stream in CONTRIBUTING.md
### Describe the bug

The link to project's zulip stream is missing in CONTRIBUTING.md. 

### To Reproduce
1. Go to [contributing.md](https://github.com/anitab-org/bridge-in-tech-backend/blob/develop/.github/CONTRIBUTING.md)
2. See error

### Expected behavior

Add link to zulip stream in CONTRIBUTING.md

### Screenshots

bit-cont

### Desktop (please complete the following information):
 - OS: Any
 - Browser: Any
 - Version: Any

### PR DESCRIPTION
**Pull request by y-martinez**
### Description

- Add the Zulip project stream link
- Improve the text changing connectors and punctuation marks

Final result

Fixes #229 

### Type of Change:

- Documentation

### Checklist:

- [X] My PR follows the style guidelines of this project
- [X] I have performed a self-review of my own code or materials
- [X] I have made corresponding changes to the documentation

**Code/Quality Assurance Only**
- [X] My changes generate no new warnings
- [X] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -9,14 +9,14 @@
...
 * In an active repository (not an archived one), choose an open issue from the issue list, claim it in the comments, and a maintainer will assign it to you.  
-* After approval you must make continuous notes on your progress in the issue while working.  If there is not at least one comment every 3 days, the maintainer can reassign the issue.
+* After approval, you must make continuous notes on your progress in the issue while working.  If there is not at least one comment every 3 days, the maintainer can reassign the issue.
 * Create a branch specific to the issue you're working on, so that you send a PR from that branch instead of the base branch on your fork.
 * If you’d like to create a new issue, please go through our issue list first (open as well as closed) and make sure the issues you are reporting do not replicate the existing issues. 
-* Have a short description on what has gone wrong (like a root cause analysis and description of the fix), if that information is not already present in the issue.
+* Have a short description of what has gone wrong (like a root cause analysis and description of the fix), if that information is not already present in the issue.
```
Comment (devkapilbansal) [inline .github/CONTRIBUTING.md]:
```suggestion
* Give a short description of what went wrong (like a root cause analysis and description of the fix), if that information is not already present in the issue.
```

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@y-martinez LGTM
Just a minor suggestion here

**Review by devkapilbansal — APPROVED** [review]
Thanks for the changes.
LGTM :+1:

**Review by epicadk — APPROVED** [review]
Looks good

**Review by decon-harsh — APPROVED** [review]

**Comment by decon-harsh** [comment]
LGTM Nice work

**Review by HeroicHitesh — APPROVED** [review]
LGTM

**Review by codesankalp — APPROVED** [review]
LGTM :rocket:

**Review by mtreacy002 — APPROVED** [review]
Well done, @y-martinez for your fix and thank you for contributing to the project 🎉🎉🎉

**Pull request merged**